src/screens/ContactScreen/index.js

```diff
@@ -37,7 +38,9 @@ const ContactScreen = ({navigation}) => {
   const [selectedUsers, setSelectedUsers] = React.useState([]);
   const [isSearchMode, setIsSearchMode] = React.useState(false);
   const [isLoadingSearchResult, setIsLoadingSearchResult] = React.useState(false);
-  const {createSignChat, loadingCreateChat} = useCreateChat();
+  const {createSignChat, createAnonymousChat, loadingCreateChat} = useCreateChat();
+  const route = useRoute();
+  const {from: sourceScreen} = route.params;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `createAnonymousChat` function is now being used, which is a good addition for handling anonymous chats. However, it's not clear from this hunk whether error handling has been properly implemented for this function. Make sure that any errors thrown by `createAnonymousChat` are caught and handled appropriately.



<!-- This is an auto-generated comment by OSS CodeRabbit -->